app/models/form_item.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# Note: set new_rank when making a form item, ignore rank on answer but set rank on response.
```

Comment:
Confused by this. FormItem doesn't have `new_rank` does it?

Reply:
I honestly do not remember what this is from. I will remove it.